src/nodeEventListeners.js

```diff
@@ -0,0 +1,26 @@
+/* --- POLYGON --- */
+
+export function onMouseDownNodePolygon() {
+    this.editEventListenerInvokeStatus("mousemove", true, this.canvas);
+    this.editEventListenerInvokeStatus("mouseup", true, this.canvas);
+}
+
+export function onMouseMoveNodePolygon() {
+    const splitIds = this.id.split("/");
```

Comment:
what does `this.id` look like that it holds a long string of shape and node ids?

Reply:
yea that's exactly what it holds. 
its used for tracking which node is being moved. it also helps update the shape appropriately